jekyll/_cci2/creating-orbs.md

```diff
@@ -13,12 +13,11 @@ Orbs are reusable packages of CircleCI configuration that you may share across p
 
 ### Orbs Quickstart
 Before you create your first orb, please see the following notes:
-* Orbs exist in namespaces, and namespaces are created & owned by organizations / Github users.
-* Namespaces are global across ALL orgs and users, so pick a unique name!
-* Namespaces cannot be removed or renamed once you have claimed a namespace.
-* Only organization owners can create namespaces.
-* By default, any orb created outside of the `circleci` namespace is an "uncertified" orb. You will need to go to your CircleCI Org Settings in the UI and enable "Allow Uncertified Orbs" in the Security section.
-* If you are creating an orb on behalf of another organization, it is recommended to use [Development Versions](#using-development-versions). Consult that section for more info.
+* Orbs exist in namespaces, and namespaces exist in a 1:1 relationship with GitHub/Bitbucket organizations/users.
+* Namespaces are global within the CircleCI orb registry, so pick a unique name!
+* Only users with "owner"/"administrator" privileges within a given GitHub/Bitbucket organization can create a namespace linked to that organization.
```

Comment:
If we're talking about permissions here it might be worth elaborating that any org member can publish dev orbs, but only admins can create new orb names or publish production orbs. And then maybe talk about how they can use API keys with admin permissions if they want to automate the process using the CLI in their jobs.